packages/builder/src/components/design/settings/controls/PhosphorIconSelect/PhosphorIconSelect.svelte

```diff
@@ -261,4 +268,8 @@
   .page-btn.disabled:hover {
     color: var(--spectrum-global-color-gray-400);
   }
+  .picker-wrap
+    :global(button.spectrum-ActionButton .spectrum-ActionButton-label) {
+    text-align: center;
```

Comment:
NAB: Why center align? Do other settings buttons with icons do that? How does this look for long icon names?

Reply:
Most of the action buttons in the settings section are centre aligned, seeing the buttons next to each other seemed off.

<img width="310" height="320" alt="Screenshot 2025-11-04 at 09 30 15" src="https://github.com/user-attachments/assets/15a45954-cc13-448f-bf3e-44cfbcb158c3" />
<img width="309" height="319" alt="Screenshot 2025-11-04 at 09 30 30" src="https://github.com/user-attachments/assets/2c183551-5424-49c8-94c4-f4c9d7443656" />

Longer icon names look grand, they truncate as expected.